src/main/kotlin/org/stellar/walletsdk/utils/sponsorOperation.kt

```diff
@@ -0,0 +1,22 @@
+package org.stellar.walletsdk.utils
+
+import org.stellar.sdk.BeginSponsoringFutureReservesOperation
+import org.stellar.sdk.EndSponsoringFutureReservesOperation
+import org.stellar.sdk.Operation
+
+fun sponsorOperation(
+  sponsorAddress: String,
+  accountAddress: String,
+  operation: Operation
+): List<Operation> {
```

Comment:
I know you're keeping it simple right now, but it might be valuable to have some validation of the operations that are passed to make sure they are sponsor-able ops.

Reply:
:100: Thanks for catching this! I added a check in [this commit.](https://github.com/stellar/kotlin-wallet-sdk/pull/3/commits/62321063c6ecaad290e70b46bfc8ab8875349f1a)